src/draive/conversation/completion/types.py

```
@@ -0,0 +1,53 @@
+
+__all__ = ("ConversationCompleting",)
+
+
+@runtime_checkable
+class ConversationCompleting(Protocol):
+    @overload
+    async def __call__(
+        self,
+        *,
+        instruction: Instruction | None,
+        input: ConversationMessage | Multimodal,
```

Pylint (3.3.7)

[warning] Redefining built-in 'input'

(W0622)